src/app.ts

```diff
@@ -1,12 +1,20 @@
 import Phaser from "phaser";
+import { DeathScene } from "./deathScene";
+import { FinishScene } from "./finishScene";
+import { LeaderboardScene } from "./leaderboard";
 import { Ntnu } from "./ntnu";
+import { PauseScene } from "./pauseScene";
+import { Uio } from "./uio";
 
-const initWidth = window.innerWidth;
-const initHeight = window.innerHeight;
+const initWidth = window.innerWidth - 15;
```

Comment:
for å ta høyde for scrollbars?
Hadde det isåfall vært bedre å finne en måte å fjerne de på?